apps/configurator/test/support/fixtures/configuration_fixtures.ex

```diff
@@ -1,7 +1,7 @@
-defmodule Configurator.ConfigurationFixtures do
+defmodule GameBackend.ConfigurationFixtures do
   @moduledoc """
   This module defines test helpers for creating
-  entities via the `Configurator.Configuration` context.
+  entities via the `GameBackend.Configuration` context.
```

Comment:
Why this change? It's inside the configurator folder, it doesn't make much sense.

Reply:
No idea why, but fixed